release_scripts/Packages.Microsoft/supported_repos_prod.json

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
same deal here - do we have to check this in?

Reply:
See my comment above. We'd need to store this data somewhere for automation to work. There isn't sensitive data so I thought a JSON would be ok.

As an alternative solution, we could store these values in a database.

Reply:
remove json, keep urls and add to script